Building-with-IntelliJ-IDEA.md

```diff
@@ -154,3 +182,9 @@ You have to add the following extra VM arguments in order to use these java vers
 ```
 --add-opens=java.desktop/com.apple.eawt=ALL-UNNAMED --add-opens=java.desktop/sun.awt=ALL-UNNAMED
 ```
+
+### WARNING: An illegal reflective access operation has occurred
```

Comment:
I think these can be moved directly at the end of conclusion with `<!-- omit in toc -->` with smaller headers, as it's not related to problems.